ugvc/comparison/vcf_pipeline_utils.py

```diff
@@ -395,7 +395,9 @@ def _fix_errors(df: pd.DataFrame) -> pd.DataFrame:
     # (FP_CA,FN_CA), (FP_CA,None) - Fake a genotype from ultima such that one of the alleles is the same (and only one)
     df.loc[(df["call"] == "FP_CA") & ((df["base"] == "FN_CA") | (df["base"].isna())), "gt_ground_truth"] = df[
         (df["call"] == "FP_CA") & ((df["base"] == "FN_CA") | (df["base"].isna()))
-    ]["gt_ultima"].apply(lambda x: ((x[0], x[0]) if (x[1] == 0) else ((x[1], x[1]) if (x[0] == 0) else (x[0], 0))))
+    ]["gt_ultima"].apply(
+        lambda x: ((x[0], x[0]) if (len(x) < 2 or (x[1] == 0)) else ((x[1], x[1]) if (x[0] == 0) else (x[0], 0)))
+    )
     return df
```

Comment:
Why did you add len(x) < 2?
Also - how is this related to geniomics.py3

Reply:
Behaviour of `bcftools norm` changed slightly in this new version and this had a weird effect downstream